metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -116,7 +116,7 @@ class Messages(icons: Icons) {
       command = ServerCommands.OpenBrowser(learnMoreUrl)
     )
     def toFixMessage =
-      "To fix this problem, upgrade to sbt v1.2.1+ or manually import the build."
+      "To fix this problem, upgrade to sbt v0.13.17+ or v1.2.1+ or manually import the build."
```

Comment:
@olafurpg do we really need 1.2.1+ or was this a legacy message? I would either update this or detect it and skip importing.